package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Oct  7 06:05:22 UTC 2019 - Knut Anderssen <kanderssen@suse.com>
+
+- AutoYaST: Added back the activation of s390 devices during the
```

Comment:
maybe add added back after what. Like `after switch to new data model`. So it is clear it is not something broken in SP1.